openapi/wicketd.json

```diff
@@ -724,6 +724,25 @@
           "message"
         ]
       },
```

Comment:
API versioning note to self / future readers: the changes made to this file are backwards incompatible, but we don't care: wicketd only serves this API on `localhost` to wicket, and they are always deployed together (since they're part of the switch zone that's included in the OS ramdisk).